### PR TITLE
remove duplicate parameter name in array multisort stub

### PR DESCRIPTION
**Pull request by terdia**
Ref: https://youtrack.jetbrains.com/issue/WI-45819

### CODE REVIEW

**Comment by andrey-sokolov** [comment]
looks like you should add a parameter back to the signature to fix the test

**Comment by terdia** [comment]
> Please, not that you have one broken tests. Looks like you should add a parameter back to the signature to fix the test. Thank you for the contribution

Thanks @andrey-sokolov updated.

**Comment by andrey-sokolov** [comment]
Thanks that was fast! Looks like the correct signature is different: https://www.php.net/manual/en/function.array-multisort.php Can you please fix it accordingly?

**Comment by terdia** [comment]
> @terdia Thanks that was fast! Looks like the correct signature is different: https://www.php.net/manual/en/function.array-multisort.php Can you please fix it accordingly?

thanks @andrey-sokolov do you mean the default values, .i.e. `$arg = 'SORT_ASC', $arg2 = 'SORT_REGULAR'`?

File: standard/standard_8.php [hunk 1]
```diff
@@ -969,25 +969,26 @@ function range ($start, $end, $step = 1) {}
 /**
  * Sort multiple or multi-dimensional arrays
  * @link https://php.net/manual/en/function.array-multisort.php
- * @param array $arr <p>
+ * @param  array  $arr  <p>
  * An array being sorted.
  * </p>
+* </p>
  * @param mixed $arg [optional] <p>
```
[the remark this answers is not in the document]
Comment (terdia) [reply standard/standard_8.php]: Great I will update the Pr later today. Thanks

**Comment by andrey-sokolov** [comment]
Sorry, I've merged this PR https://github.com/JetBrains/phpstorm-stubs/pull/686 for this issue as ready one. Thank you for participating

**Pull request closed without merging**